torch/csrc/autograd/python_variable.cpp

```diff
@@ -809,8 +868,180 @@ static PyMethodDef extra_methods[] = {
   {nullptr}
 };
 
+#define DEFERRED_ADDRESS(ADDR) nullptr
+
+struct THPVariableMeta {
+  PyTypeObject base;
+  // TODO: there are more fields for GC shenanigans
```

Comment:
What are the other fields? Is this comment still accurate?

Reply:
Yeah, it is. I'm not sure what the other fields are, I just know that when I specify `tp_basicsize` as `sizeof(THPVariableMeta)` I cause a GC bug. Put this one on my TODO list; I just need to go and look at what `PyType_Type` struct actually is.

Reply:
It turns out I'm supposed to use PyHeapTypeObject